pages/ppr-marketing.vue

```diff
@@ -1,68 +1,68 @@
-            </v-col>
-          </v-row>
-
-          <v-row v-if="!loggedIn" class="pt-6" no-gutters>
-            <v-col class="body-text">
-              New to BC Registries?
-              <a class="font-weight-bold" :href="$config.authURL + 'choose-authentication-method'">
-                Create a BC Registries Account
-              </a>
-            </v-col>
-          </v-row>
-
-          <v-row v-if="loggedIn" class="pt-6" no-gutters>
-            <v-col>
-              <v-btn
-                  large
-                  class="button-login font-weight-bold mr-2"
-                  href="/dashboard"
-                >
-                  Go to BC Registries Dashboard
-                </v-btn>
-            </v-col>
-          </v-row>
-        </v-container>
-      </header>
-    </article>
+    <header class="ppr-bg auth d-flex align-center">
+      <v-container>
+        <v-row no-gutters>
+          <v-col class="col-md-6 col-sm-12">
+            <h1 class="text-left bigger-h1">
+              Register or search for legal claims on personal property
+            </h1>
+          </v-col>
+        </v-row>
+
+        <v-row no-gutters>
+          <v-col class="col-md-5 col-sm-12 body-text">
+            The Personal Property Registry records security interests and
+            liens against personal property belonging to British Columbia businesses
+            and individuals.
+          </v-col>
+        </v-row>
+
+        <v-row v-if="!loggedIn" class="pt-6" no-gutters>
+          <v-col class="col-12 col-md-6">
+            <SbcSigninButton />
+          </v-col>
+        </v-row>
+
+        <v-row v-if="!loggedIn" class="py-6" no-gutters>
+          <v-col class="body-text">
+            New to BC Registries?
+            <a
+              class="font-weight-bold"
+              :href="$config.authURL + 'choose-authentication-method'"
+            >
+              Create a BC Registries Account
+            </a>
+          </v-col>
+        </v-row>
+
+        <v-row v-if="loggedIn" class="py-6" no-gutters>
+          <v-col>
+            <v-btn
+              large
+              class="button-white-on-blue"
+              href="/dashboard"
+            >
+              Go to BC Registries Dashboard
+            </v-btn>
+          </v-col>
+        </v-row>
+      </v-container>
+    </header>
 
     <v-container class="body-text">
-      <h1 class="text--center" style="padding-top: 80px">Coming Soon...</h1>
+      <h1 class="text-center" style="padding-top: 80px">Coming Soon...</h1>
 
       <v-row class="pt-3 pb-8">
-        <v-col cols="12" class="text--center">
+        <v-col cols="12" class="text-center">
           In early 2022, the BC Personal Property Registry will be launching an
           all-new application.
         </v-col>
       </v-row>
 
       <v-row style="padding-bottom: 80px">
-        <v-col class="text--center px-6 col-md-3 col-sm-6">
+        <v-col class="text-center px-6 col-12 col-md-3">
```

Comment:
"By default, be 12 columns wide. From MD and greater, be 3 columns wide."

This allows nice responsive (mobile) layout.